### PR TITLE
Add status messages for connection status

### PR DESCRIPTION
**Pull request by rudigiesler**
We should be looking for the cases when a connection up/down event is sent in yowsup, and send the appropriate status event.

### CODE REVIEW

**Comment by rudigiesler** [comment]
Ready for review.

File: vxyowsup/whatsapp.py [hunk 1]
```diff
@@ -249,6 +259,13 @@ def onAck(self, ack):
         '''receives confirmation of delivery to server'''
         # sent_message_id: whatsapp id
         # user_message_id: vumi_id
-        log.info('Received %s' % ' '.join(str(ack).split('\n')))
+        self.transport.log.info('Received %s' % ' '.join(str(ack).split('\n')))
         if ack.getClass() == "message":
             reactor.callFromThread(self.transport._send_ack, ack.getId())
+
+    def onEvent(self, event):
+        if event.getName() == YowNetworkLayer.EVENT_STATE_CONNECTED:
```
Comment (hodgestar) [inline vxyowsup/whatsapp.py]: Should we perhaps only get the name once and use `elif`?

**Comment by hodgestar** [comment]
Left one minor comment, otherwise looks good.

Woot on fixing logging.

**Comment by rudigiesler** [comment]
@hodgestar Comments addressed. Ready for another review.

**Comment by hodgestar** [comment]
:+1:

**Pull request merged**